src/ServicePulse.Host/app/js/views/archive/view.html

```diff
@@ -11,7 +11,7 @@
         <div class="row" ng-show="!vm.loadingData">
             <div class="col-sm-12">
                 <div class="btn-toolbar">
-                    <button type="button" class="btn btn-default" confirm-message="Are you sure you want to un-archive selected archived messages?" confirm-click="vm.unarchiveSelected()" ng-disabled="vm.selectedIds.length == 0"><i class="fa fa-undo"></i> Unarchive {{vm.selectedIds.length | number}} selected</button>
+                    <button type="button" class="btn btn-default" confirm-title="Are you sure you want to un-archive selected archived messages?" confirm-message="The messages will be visible on the failed message screen." confirm-click="vm.unarchiveSelected()" ng-disabled="vm.selectedIds.length == 0"><i class="fa fa-undo"></i> Unarchive {{vm.selectedIds.length | number}} selected</button>
```

Comment:
`Unarchived messages will be moved back to the list of failed messages.`

@WojcikMike in addition to this, are there any potential consequences for queues, endpoints or the system in general?